docs/terms/expectation.md

```diff
@@ -3,10 +3,134 @@ title: Expectation
 id: expectation
 hoverText: A verifiable assertion about data.
 ---
+import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
+import TechnicalTag from '../term_tags/_tag.mdx';
+import SetupHeader from '/docs/images/universal_map/_um_setup_header.mdx'
+import ConnectHeader from '/docs/images/universal_map/_um_connect_header.mdx';
+import CreateHeader from '/docs/images/universal_map/_um_create_header.mdx';
+import ValidateHeader from '/docs/images/universal_map/_um_validate_header.mdx';
+
+
+<UniversalMap setup='inactive' connect='inactive' create='active' validate='active'/> 
+
+## Overview
+
+### Definition
 
 An Expectation is a verifiable assertion about data. 
 
+### Features and promises
+
+Great Expectations is a framework for defining Expectations and running them against your data. Like assertions in traditional Python unit tests, Expectations provide a flexible, declarative language for describing expected behavior. Unlike traditional unit tests, Great Expectations applies Expectations to data instead of code. For example, you could define an Expectation that a column contain no null values, and Great Expectations would run that Expectation against your data, and report if a null value was found.
```

Comment:
```suggestion
Great Expectations is a framework for defining Expectations and running them against your data. Like assertions in traditional Python unit tests, Expectations provide a flexible, declarative language for describing expected behavior. Unlike traditional unit tests, Great Expectations applies Expectations to data instead of code. For example, you could define an Expectation that a column contains no null values, and Great Expectations would run that Expectation against your data, and report if a null value was found.
```